openedx/features/classrooms/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from models import DiscussionCommunityMembership
+
+
+class DiscussionCommunityMembershipAdmin(admin.ModelAdmin):
+    list_display = ('user', 'community')
```

Comment:
Let's consider adding the course in there, too, for better visualisation. Also please have a look at #1456, that fix must be followed to avoid timeout errors again.

Reply:
Done.